client/src/js/components/bhAddItem.js

```diff
@@ -0,0 +1,35 @@
+angular.module('bhima.components')
+  .component('bhAddItem', {
+    templateUrl : 'modules/templates/bhAddItem.tmpl.html',
+    controller  : addItemController,
+    transclude  : true,
+    bindings    : {
+      disable          : '<',
+      callback         : '&?',
```

Comment:
This should be required (`&` instead of `&?`).  The module will break without it!